perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/MembersManagerMethod.java

```diff
@@ -1122,18 +1122,20 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 	/*#
 	 * Set membership status of a member.
 	 *
+	 * @deprecated use vosManager setBan to suspend a member
+	 *
 	 * @param member int Member <code>id</code>
-	 * @param status String VALID | INVALID | SUSPENDED | EXPIRED | DISABLED
-	 * @exampleParam status "SUSPENDED"
+	 * @param status String VALID | INVALID | EXPIRED | DISABLED
+	 * @exampleParam status "VALID"
 	 * @param message String reason for suspension
```

Comment:
If ``setStatus()`` method version with ``message`` param was removed, we should remove its docs completely.

Reply:
removed